common-graphql/src/main/resources/graphql/schemas/UserPreferencesDB.graphql

```diff
@@ -344,6 +344,7 @@ input ExplorePlotTimeUpdates {
 columns and relationships of "exploreTargetPreferences"
 """
 type ExploreTargetPreferences {
+  acquisitionOffsets: Boolean
```

Comment:
Is this temporary while they are not in the API?

Reply:
This a preference as to whether to display them or not, not related to the model